### ISSUE TITLE
Dynamic Loops Styling

### ISSUE THREAD

**Issue by inregpartners** [dudaster/ele-custom-skin#262]
I have a handful of Elementor Sections (templates) that leverage various Loops. Each of these loops build Ads that are incorporated into my templates via shortcodes. Each Loop consists of dynamic elementor elements that map to ACF fields for an Advertisements CPT.

There are various Ad layouts (single; dual; triple) that pull in background images, headings, etc. -- all of which are dynamically driven by ACF fields. I have a wierd anomaly that doesn't occur in Dev version of site where I lose some styling (padding, margins, etc.) of a Loop after I save an update to another Loop.

i.e. A specific Loop looks as expected immediately after I save changes in Elementor for the loop. As soon as I save changes for another loop, the previously OK Loop now has lost some of its styling/layout. Essentially has limited me to only be able to have one Loop that will render properly.

I have other Loop types that don't seem to be affected. It's just these six Ads loops that seem to be conflicting with each other for some reason. See attached for reference.
[example.pdf](https://github.com/dudaster/ele-custom-skin/files/4869685/example.pdf)

**Comment by dudaster** [comment]
please provide an url so I can inspect element

**Comment by inregpartners** [comment]
<https://hipparis.com/ads/>https://hipparis.com/ads/

Thanks

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by dudaster** [comment]
Have you tried setting the section column gap to no gap?

**Comment by inregpartners** [comment]
Yes, and it works but only for the last Loop that was saved. As soon as I save a different Loop the one that was previously styled correctly loses some of the styling. It looks to be just 0 margin / 0 padding settings that are affected.

There are six different Loops for the various Ad types, so that means 5 of the six lose some of their styling.

**Comment by dudaster** [comment]
So each section in the article should have a  .elementor-column-gap-no class.

For this to happen you need to edit the Loop Template and the main section select column cap to no gap.

**Comment by inregpartners** [comment]
Great. Thank you very much for the assistance!
